test/verify/check-system-services

```diff
@@ -630,10 +630,18 @@ WantedBy=multi-user.target
         b.wait_not_present("#condition")
 
     def testRelationships(self):
+        self._testRelationships()
+
+    def testRelationshipsSession(self):
```

Comment:
Nitpick: Please let's call this "RelationshipsUser" for consistency -- nothing here applies to a "session", neither in the logind nor in the cockpit sense.